lib/battlestation/cli.rb

```diff
@@ -133,45 +133,24 @@ def install_packages(current_dirname)
         'ios-sim',
         'python',
         'rbenv',
-        'terminal-notifier',
-        'youtube-dl',
         'gh',
       ]
-      # Slowly bringing code into ruby from the monolithic setup script.
-      system 'bash', '-c', %{
-#######################################
-# Prints the provided message to STDOUT in green.
-#
-# Arguments:
-#   Info message
-#######################################
-echo_info() {
-  printf "$(tput setaf 2)%s$(tput sgr 0)\n" "$*" >&2;
-}
-
-#######################################
-# Installs the specified Homebrew package if it isn't installed. If it is, tries to upgrade the
-# package.
-#
-# Arguments:
-#   Name of package
-#######################################
-install_or_upgrade_package() {
-  if ! #{full_homebrew_path} list "$1" >/dev/null 2>&1; then
-    echo_info "Installing $1 via Homebrew"
-    #{full_homebrew_path} install "$1"
-  else
-    # TODO: find a way to only update if needed to prevent errors in console.
-    #{full_homebrew_path} upgrade "$1"
-  fi
-}
 
-packages=( #{packages.join(" ")} )
-for package in "${packages[@]}"
-do
-  install_or_upgrade_package "$package"
-done
-      }
+      for package in packages
+        if system(full_homebrew_path, 'list', package, out: File::NULL, err: File::NULL)
+          # Package is installed. Only update it if it's outdated.
+          outdated = system(full_homebrew_path, 'outdated', package, out: File::NULL, err: File::NULL)
```

Comment:
It's not straightforward to install an older package version so I wasn't able to test this code path.

Reply:
I ran the setup script on another computer that had outdated packages and I see them being updated 👍